src/datagrid/DataGrid.tsx

```diff
@@ -1098,6 +1282,85 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
+                ? row.detailPaneData
+                : {
+                      title: undefined,
+                      detailPaneContent: undefined,
+                      isOpen: false,
+                  };
+            const showDetailPane: boolean = isOpen && detailPaneContent;
+
+            return (
+                <React.Fragment>
+                    {showDetailPane && (
+                        <div
+                            _ngcontent-byx-c249=""
+                            className={classNames([
+                                ClassNames.DATAGRID_DETAIL_PANE,
+                                ClassNames.DATAGRID_NG_STAR_INSERTED,
+                            ])}
+                        >
+                            <span
+                                tabIndex={0}
+                                className={classNames([
+                                    ClassNames.OFFSCREEN_FOCUS_REBOUNDER,
+                                    ClassNames.DATAGRID_NG_STAR_INSERTED,
+                                ])}
+                            />
+                            <div
+                                role="dialog"
+                                aria-modal="true"
+                                className={classNames([
+                                    ClassNames.DATAGRID_DETAIL_PANE_CONTENT,
+                                    ClassNames.DATAGRID_NG_STAR_INSERTED,
+                                ])}
+                                id={`${"clr-id-"}-${rowID}`}
+                                aria-describedby={`clr-id-${rowID}-title`}
+                                tabIndex={-1}
+                            >
+                                <div className={ClassNames.CLR_SR_ONLY} />
+                                <div _ngcontent-byx-c249="" className={ClassNames.DATAGRID_DETAIL_HEADER}>
+                                    <div
+                                        className={ClassNames.DATAGRID_DETAIL_HEADER_TITLE}
+                                        id={`clr-id-${rowID}-title`}
+                                    >
+                                        {title}
+                                    </div>
+                                    <div className={ClassNames.DATAGRID_DETAIL_PANE_CLOSE}>
+                                        <Button
+                                            aria-label="Close"
+                                            className={ClassNames.PAGINATION_NEXT}
+                                            icon={{shape: "close"}}
+                                            onClick={() => rowID !== undefined && this.handleDetailPaneToggle(rowID)}
+                                        />
+                                    </div>
+                                </div>
+                                {/* close header*/}
+                                <div ngcontent-byx-c249="" className={ClassNames.DATAGRID_DETAIL_BODY}>
```

Comment:
why is empty content needed > 
 **ngcontent-byx-c249=""**

Reply:
removed